aio/deploy/helm-chart/kubernetes-dashboard/templates/clusterrole-metrics.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if .Values.rbac.clusterRoleMetrics }}
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: "{{ template "kubernetes-dashboard.fullname" . }}"
```

Comment:
@desaintmartin I'm trying to wrap my head around why an additional clusterrole and clusterrolebinding were added with their own settings. This conflicts (it's identical to) what's in clusterrole.yaml and attempts to deploy the latest here now fail because with rbac.create=true and rbac.clusterRoleMetrics=true the chart is attempting to create duplicate clusterrole and clusterrolebinding resources of the same name.

```
clusterroles.rbac.authorization.k8s.io "kubernetes-dashboard" already exists
```

Were these changes related to the PR I opened with support for the dashboard-metrics-scraper or something else?

Reply:
I've just renamed clusterrole.yaml to clusterrole-metrics.yaml and clusterrolebinding.yaml to clusterrolebinding-metrics.yaml, and be more modular about which one(s) you want.

Previously, you had to install all 4 at once (all or nothing).

So this must be related to to this change. Does this resource still exist if you uninstall the release? I am not able to reproduce on my cluster (which does not mean it is not my fault!).

Reply:
Ok, so this was totally my fault. Had used `cp -R` to copy in changes from the updated PR here, and ended up with both the old files _and_ the new ones in my local copy of the chart.

Sorry for the false alarm! 🙈